README.md

```diff
@@ -231,6 +234,17 @@ const credentialsAuthenticator = createAuthenticator({
   always returns a rejected promise resulting in an unauthenticated session. It
   is important that this function is defined when creating your authenticator.
 
+* `invalidate(data)` (_function_): A function responsible for doing any 
+  additional cleanup of the authenticated data. This function will be invoked 
+  when the [`invalidateSession`](#invalidatesession) action is dispatched. It
+  accepts a single argument of data and must return a promise. A resolved 
+  promise will result in the session becoming unauthenticated. A rejected 
+  promise will result in invalidation being interrupted, however session data
```

Comment:
Hmm yeah, i do agree that aligning with the Ember paradigm is probably more valuable than having some quirkiness because of how redux might solve the problem. I'll loop back to this.